tests/components/sensor/test_wunderground.py

```diff
@@ -292,3 +366,27 @@ def test_invalid_data(self, req_mock):
         for device in self.DEVICES:
             device.update()
             self.assertEqual(STATE_UNKNOWN, device.state)
+
+    @unittest.mock.patch('requests.get',
+                         side_effect=mocked_requests_get_alerts)
+    @unittest.mock.patch(
+        'homeassistant.components.sensor.wunderground.WUndergroundData.update',
+        side_effect=partial(
+            wunderground.WUndergroundData.update,
+            no_throttle=True))
+    def test_alert_data(self, req_mock, upd_mock):
+        """Test the WUnderground invalid data."""
+        wunderground.setup_platform(self.hass, VALID_CONFIG_ALERT,
+                                    self.add_devices, None)
+        for device in self.DEVICES:
+            device.update()
+            device.update()
```

Comment:
`device.update(no_throttle=True)` will skip the Throttle.

Reply:
I tried that in f7e0b0271297f8c63eb9eaa15ecdc8baf5db5552 but that was not the correct level, sadly. Hopefully, I get around to convert this PR to the new tests and check if this works now.